fe/src/main/java/org/apache/doris/transaction/TransactionState.java

```diff
@@ -188,6 +197,8 @@ public void write(DataOutput out) throws IOException {
         for (long errorReplciaId : errorReplicas) {
             out.writeLong(errorReplciaId);
         }
+        // TODO(ml): extra write
+        Text.writeString(out, extra.toString());
```

Comment:
Don't use toString to serialize. If you want to serialize it, you should add `write` function to `TxnStateExtra`